### ISSUE TITLE
[Feature Request] - Optional logging to stdout for GELF \ Graylog

### ISSUE THREAD

**Issue by atxbyea** [linuxserver/docker-swag#215]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

Wish for an option to pipe all logs to stdout\stderr to be able to use this content pack with graylog
https://github.com/lewisgeorge-innoscale/graylog3-content-pack-nginx-docker

------------------------------

## Desired Behavior

A flag to be able to redirect logs to stdout\stderr if wanted
Basically just do 
ln -sf /dev/stdout /var/log/nginx/access.log && ln -sf /dev/stderr /var/log/nginx/error.log

## Current Behavior

The docker logs are empty, no logs displayed after startup is complete

## Alternatives Considered

I have edited the container while running to create the symbolic links, and it works

**Comment by Roxedus** [comment]
Nginx supports writing to syslog http://nginx.org/en/docs/http/ngx_http_log_module.html#access_log and http://nginx.org/en/docs/ngx_core_module.html#error_log. 

One can also use the universal mod that tails a log stdout https://github.com/linuxserver/docker-mods/tree/universal-stdout-logs.
